packages/cactus-test-tooling/package.json

```diff
@@ -82,6 +82,7 @@
   "homepage": "https://github.com/hyperledger/cactus#readme",
   "dependencies": {
     "@hyperledger/cactus-common": "0.5.0",
+    "@types/js-yaml": "4.0.1",
```

Comment:
@RafaelAPB Recommend moving this to the `devDependencies` section because the type information package is not needed at runtime (because Typescript at runtime is just Javascript)

```suggestion
```